src/components/Panel/PopoverPanel.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+import { config } from 'storybook-addon-designs';
+import { useState } from '@storybook/addons';
+
+import { StoryPage } from '@sb/StoryPage';
+
+import { PanelVariant } from '../Panel';
+import { Button } from '../Button';
+
+import { PopoverPanel, PopoverSide } from './PopoverPanel';
+
+const figmaLink = ''; // TODO: Add figma link
+
+export default {
+  title: `Components/Panel/PopoverPanel`,
+  component: PopoverPanel,
+  parameters: {
+    design: config([
+      {
+        type: 'figma',
+        url: figmaLink,
+      },
+      {
+        type: 'link',
+        url: figmaLink,
+      },
+    ]),
+    docs: {
+      page: () => (
+        <StoryPage
+          description={`TODO: Add a description (supports markdown)`}
+        />
+      ),
+    },
+  },
+  args: {
+    title: 'Tittel',
+    variant: PanelVariant.Warning,
+    trigger: <button>Åpne</button>,
+    side: PopoverSide.Top,
+  },
+} as ComponentMeta<typeof PopoverPanel>;
+
+const Template: ComponentStory<typeof PopoverPanel> = (args) => {
+  const [open, onOpenChange] = useState(false);
+
+  const handleOnOpenChange = () => {
+    onOpenChange(!open);
```

Comment:
I would prefer to rename this function - its not an event to react to, which the current name suggests. The common naming pattern for useState is `const [someValue, setSomeValue] = useState(initialValue)`, because the function is just updating (setting) a value.

```suggestion
  const [open, setOpen] = useState(false);

  const handleOnOpenChange = () => {
    setOpen(!open);
```